en_US.json

```diff
@@ -823,11 +823,15 @@
 	"error.origin.authentication.awss3.secretkey.length": "The AWS S3 secret key must be at most 4096 characters.",
 	"error.origin.auto.detect.popup.msg": "We could not detect a server based on your hostname. Please manually enter origin settings.",
 	"error.origin.auto.detect.popup.title": "Auto Detection Failed",
+	"error.origin.failureThresholdLabel.range": "The Failure Threshold value must be in the range [1, 999]",
+	"error.origin.failureThresholdLabel.required": "Failure Threshold value is required",
 	"error.origin.invalid.domain.name": "Must be a valid domain name or a variable starts with '$'.",
 	"error.origin.invalid.keepalive.timeout": "The Keep-Alive Timeout value must be in the range [5, 600]",
 	"error.origin.invalid.tlsSessionTimeoutDefault": "Invalid TLS Session Timeout value.",
 	"error.origin.invalid.tlsSessionTimeoutRange": "The TLS Session Timeout value must be in the range [300, 86400]",
 	"error.origin.select.required": "Please select an origin.",
+	"error.origin.timeout.range": "The Peer Failure Timeout value must be in the range [1, 1200]",
```

Comment:
"The Timeout value must be in the range"  since this field is a part of the peerFailureTimeout setting.

Reply:
@chiraags I have made the changes in the new commit.